erpnext/stock/doctype/item/item.py

```diff
@@ -635,8 +637,8 @@ def validate_name_with_item_group(self):
 
 	def update_item_price(self):
 		frappe.db.sql("""update `tabItem Price` set item_name=%s,
-			item_description=%s, brand=%s where item_code=%s""",
-					(self.item_name, self.description, self.brand, self.name))
+			item_description=%s, brand=%s, uom=%s where item_code=%s""",
+					(self.item_name, self.description, self.brand, self.name, self.stock_uom))
```

Comment:
Positional values must be in the right order
```suggestion
					(self.item_name, self.description, self.brand, self.stock_uom, self.name))
```

Reply:
Just use named arguments instead? keeping track of 5 arguments in order is bound to be error prone. @18alantom